### PR TITLE
[Auth] #394 SIWE messge formatter

### PR DESCRIPTION
**Pull request by llbartekll**
add SIWE message formatter

### CODE REVIEW

File: Sources/Auth/Services/Common/SIWEMessageFormatter.swift [hunk 1]
```diff
@@ -1,11 +1,91 @@
...
+                URI: \(uri)
+                Version: \(version)
+                Chain ID: \(chainId)
+                Nonce: \(nonce)
+                Issued At: \(iat)\(expLine)\(nbfLine)\(requestIdLine)\(resourcesSection)
+                """
+    }
+
+    var expLine: String {
+        if let exp = exp {
```
Comment (flypaper0) [inline Sources/Auth/Services/Common/SIWEMessageFormatter.swift:54]: may be guard? Here and in other params

File: Sources/Auth/Services/Common/SIWEMessageFormatter.swift [hunk 2]
```diff
@@ -1,11 +1,91 @@
...
+            return "\nRequest ID: \(requestId)"
+        }
+        return ""
+    }
+
+    var resourcesSection: String {
+        var section = ""
+        if let resources = resources {
+            section = "\nResources:"
+            resources.forEach {
```
Comment (flypaper0) [inline Sources/Auth/Services/Common/SIWEMessageFormatter.swift:85]: good to use reduce here or join(separator: )

Comment (llbartekll) [reply Sources/Auth/Services/Common/SIWEMessageFormatter.swift]: ok it is shorter

**Review by flypaper0 — CHANGES_REQUESTED** [review]
need to make work testNilOptionalParamsMessage test

**Review by flypaper0 — APPROVED** [review]

**Pull request merged**